CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ if (${REST_MPFR} MATCHES "ON")
     find_path(MPFR_INCLUDE mpfr.h HINTS ${MPFR_PATH} ${MPFR_PATH}/include) # path to include directory
     find_path(MPFR_LIB_PATH libmpfr.so HINTS ${MPFR_PATH} ${MPFR_PATH}/lib)
     find_library(MPFR_LIB mpfr HINTS ${MPFR_PATH} ${MPFR_PATH}/lib) # path to .so file
+	add_definitions(-DUSE_MPFR)
```

Comment:
Don't understand the change, commit suggestion, why the PR is blocked? This was just coming from #326. But since it is a minor update I guess is not a big deal.

The merge seems to be blocked now.

Reply:
In my opinion you should either address my comment or remove this change from this PR and discuss it in the appropiate PR. As I raised a question in https://github.com/rest-for-physics/framework/pull/326 which was unanswered. If you want to merge the PR, just remove the cmake change from this PR (which should be documentation only anyway) and we can discuss it in a more appropiate PR.

Reply:
I don't see the question at #326

Reply:
> I don't see the question at #326

Since I cannot quote the comment (only the PR) I will screenshot:

![image](https://user-images.githubusercontent.com/35803280/202655335-abafaa1b-1fc2-4d17-8ba4-00f51720c6b1.png)

Reply:
I see nothing on my side

<img width="1301" alt="Screenshot 2022-11-18 at 09 31 36" src="https://user-images.githubusercontent.com/12447509/202657007-6767485a-46ce-4015-8a6f-a7d644d882ee.png">
<img width="1327" alt="Screenshot 2022-11-18 at 09 31 49" src="https://user-images.githubusercontent.com/12447509/202657031-562f1042-a2e7-4bdc-9392-cd237e083845.png">

Reply:
I understand nothing, I cannot see your message where you state " ... called in the `if` below, so probably your problem is elsewhere?". 

Which problem?

Reply:
Okay this explains the confusion! I updated your original PR in [9e3315b](https://github.com/rest-for-physics/framework/pull/326/commits/9e3315b4f5ff7a5b2472bf1be8f8375b8811c8e7), since it duplicated a definition and also used a different syntax. It should have the same funtionality.